### ISSUE TITLE
Implement LogIn/Sign Up

### ISSUE THREAD

**Issue by shreya024** [shreya024/MemoriesApp#5]
Implement the functionality of allowing users to login / sign up by choosing options from popular social media logins.

**Comment by KARTIK-RAINA5802** [comment]
Heyy can i implement this functionality as i am familiar with node js, express and Oauth 2.0.

**Comment by souravbhunia07** [comment]
Heyy can I implement this functionality. please assign it to me.

**Comment by fykaa** [comment]
Hey @souravbhunia07  this issue is already assigned, we will be coming up with new issues, You can contribute to them. All the best 🎉

**Comment by KARTIK-RAINA5802** [comment]
Hey, do I need to create a frontend for Login/Sign Up also or just implementation, because there is no page for the login or sign up part and I could see someone is assigned to create a login page and which Social media login do we need for the user to sign in ?

**Comment by shreya024** [comment]
> Hey, do I need to create a frontend for Login/Sign Up also or just implementation, because there is no page for the login or sign up part and I could see someone is assigned to create a login page and which Social media login do we need for the user to sign in ?

I have just merged the frontend for the login page. U can just work on implementation ,but feel free to make any minor changes required for the implementation

**Comment by KARTIK-RAINA5802** [comment]
Yess i just pulled the changes. Can you also merge the Sign Up page.

**Comment by shreya024** [comment]
> Yess i just pulled the changes. Can you also merge the Sign Up page.

It is the Login/Sign Up page that I have merged.If you need major changes, let me know and I'll create another issue.Else you could implement using Conditional Rendering of text and minor changes as necessary

**Comment by shreya024** [comment]
@KARTIK-RAINA5802 Any updates?

**Comment by KARTIK-RAINA5802** [comment]
Yes actually this is my first time with React js and I was trying to get that sign-up page but ran into a lot of errors but finally solved that now and it's working fine. 

Now I will start with the implementation part.

**Comment by KARTIK-RAINA5802** [comment]
Hey, I have implented the login/sign up functionality and also used passport.js to add authentication to make user credentials more secure and it's working fine. 

May I make a PR ?

**Comment by shreya024** [comment]
Sure go ahead @KARTIK-RAINA5802

**Comment by KARTIK-RAINA5802** [comment]
hey, I tried to make a PR (actually 3) and it is able to pass two checks but fails on GitGuardian Security Checks. It's related to some mongoDB credentials so I removed the connection URL and commented secret part but it's still not happening. This is my first time making a PR and I don't know much about it. Can you help me with this issue?

**Comment by tharani-2006** [comment]
I’d like to work on this issue. Could you please assign it to me?
